tests/model/ImageTest.php

```diff
@@ -69,29 +69,44 @@ public function tearDown() {
 	}
 
 	public function testGetTagWithTitle() {
+		$origForceResample = Config::inst()->get('Image', 'force_resample');
```

Comment:
You don't need to do this in tests anymore. `Config` and `Injector` are sand-boxed for each test